src/main/java/io/rpg/model/actions/ActionEngine.java

```diff
@@ -131,7 +131,9 @@ public void onAction(BattleAction action) {
   }
 
   public void onAction(LevelUpAction action) {
-    controller().getPopupController().openTextPopup((action.getPoints() != null ? (action.getPoints() + " points earned already!\n") : "" ) + "Achieved level " + action.newLevel + "!",
+    controller().getPopupController().openTextPopup(
+        (action.getPoints() != null ? (action.getPoints() + " points earned already!\n") : "")
+            + "Achieved level " + action.newLevel + "!",
```

Comment:
I think it'll look better if we change all of the popup texts to Polish to match the demo configuration.

Reply:
IMO in separate PR